### DISCUSSION TITLE
Speedometer with ESP32-2432S028R V3 (2USB) and  NEO-6M-GPS or  ATGM336H GPS

### DISCUSSION THREAD

**Discussion by stevedee78** [witnessmenow/ESP32-Cheap-Yellow-Display#263] [General]
Can someone help me with my code? I am trying to display GPS data on the Cheap Yellow Display. 
On the display it also shows me the number of satellites but for Speed:, Lat: and Lng: it only shows me an ‘f’. The code was written with Arduino IDE. Maybe someone can test it. Serial GPIOS are 17 and 16. 

The code is attached.

[CYD_Tacho.ino.txt](https://github.com/user-attachments/files/17297450/CYD_Tacho.ino.txt)

**Comment by stevedee78** [comment]
I have solved. https://github.com/stevedee78/CYD_GPS_Speedometer

**Reply by tetris338** [reply]
Hello
Could you point me to the libraries for this project and their versions? What Arduino IDE did you use?
